koku-metrics-operator/3.0.1/manifests/koku-metrics-operator.clusterserviceversion.yaml

```diff
@@ -51,6 +51,10 @@ metadata:
     support: Cost Management
   name: koku-metrics-operator.v3.0.1
   namespace: placeholder
+  labels:
+    operatorframework.io/arch.amd64: supported
+    operatorframework.io/arch.ppc64le: supported
+    operatorframework.io/os.linux: supported
```

Comment:
This file should not be updated because this is already a published version of the operator. (We will not be updating old versions of the operator to support new architectures)

We should add these here instead:
https://github.com/project-koku/koku-metrics-operator/blob/main/config/manifests/bases/koku-metrics-operator.clusterserviceversion.yaml#L3
so that future versions will contain these labels.